src/api/app/models/workflow/step/branch_package_step.rb

```diff
@@ -9,6 +11,14 @@ def call(options = {})
     branch_package(workflow_filters)
   end
 
+  def source_project_name
+    step_instructions[:source_project]
+  end
+
+  def target_project_name
+    "home:#{@token.user.login}:#{source_project_name}:PR-#{scm_webhook.payload[:pr_number]}"
```

Comment:
I don't really like that this is duplicated now, because this code is going to change in our current deliverable and the logic will be duplicated in 3 steps.

Reply:
Ok, I can move it back to Workflow::Step and overwrite it inside ConfigureRepositoriesStep.

Reply:
done